draft-ietf-rtcweb-mdns-ice-candidates.md

```diff
@@ -104,8 +104,8 @@ are not exposed to the web application.
 Principle {#principle}
 ============
 
-This section uses the concept of ICE agent as define in {{RFC8445}}.
-In the remainder of the document, it is assumed that each browser execution context has its own ICE agent.
+This section uses the concept of ICE agent as defined in {{RFC8445}}. In the
+remainder of the document, it is assumed that each browsing context has its own ICE agent.
```

Comment:
```suggestion
remainder of the document, it is assumed that each [browsing context](https://html.spec.whatwg.org/multipage/browsers.html#browsing-context) has its own ICE agent.
```

Reply:
Alas, you'll need to ref this IETF-style, similar to how WebRTCSpec is linked.